test/core/rebalancingSetToken.spec.ts

```diff
@@ -128,6 +130,8 @@ contract('RebalancingSetToken', accounts => {
       subjectInitialUnitShares = DEFAULT_UNIT_SHARES;
       subjectProposalPeriod = ONE_DAY_IN_SECONDS;
       subjectRebalanceInterval = ONE_DAY_IN_SECONDS;
+      subjectEntranceFee = ZERO;
```

Comment:
Might be a bit of a nit but feel like these should be initialized with a non-zero value, because if nothing is assigned to the state var on the smart contract I believe it'll return 0. So I think to avoid an unlikely, though possible regression (the code looks fine) we should make this a positive number.

Reply:
+1